apps/meteor/tests/end-to-end/api/01-users.js

```diff
@@ -3105,33 +3154,51 @@ describe('[Users]', function () {
 					})
 					.end(done);
 			});
+
+			it('should return users that are subscribed to the same rooms as the requester', (done) => {
+				joinChannel(user2Credentials, roomId).then(() => {
+					request
+						.get(api('users.autocomplete?selector={}'))
+						.set(userCredentials)
+						.expect('Content-Type', 'application/json')
+						.expect(200)
+						.expect((res) => {
+							expect(res.body).to.have.property('success', true);
+							expect(res.body).to.have.property('items').and.to.be.an('array').with.lengthOf(1);
+						})
+						.end(done);
+				});
+			});
```

Comment:
pls prefer async/await:
```suggestion
			it('should return users that are subscribed to the same rooms as the requester', async () => {
				await joinChannel(user2Credentials, roomId);
				request
					.get(api('users.autocomplete?selector={}'))
					.set(userCredentials)
					.expect('Content-Type', 'application/json')
					.expect(200)
					.expect((res) => {
						expect(res.body).to.have.property('success', true);
						expect(res.body).to.have.property('items').and.to.be.an('array').with.lengthOf(1);
					});
			});
```